### PR TITLE
Improve app layout navigation

### PR DESCRIPTION
**Pull request by andresgutgon**
# What?
Improve app navigation

## TODO
- [x] Validate email uniqueness 
- [x] Apply changes in this issue https://github.com/latitude-dev/latitude-llm/issues/177

### CODE REVIEW

File: apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-        },
-        {
-          name: <Text.H5M>Evaluations</Text.H5M>,
-        },
-      ]}
-    >
-      {children}
-    </AppLayout>
-  )
-}
```
Comment (geclos) [inline apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx:38]: is this breaking any modals?

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/_components/BreadcrumpInput.tsx [hunk 2]
```diff
@@ -21,7 +24,9 @@ export default function BreadcrumpInput({ projectId }: { projectId: number }) {
     { trailing: true },
   )
 
-  if (!project) return null
+  if (!project) {
+    return <Text.H5M color='foregroundMuted'>{projectName}</Text.H5M>
+  }
 
   return (
     <EditableText
```
Comment (geclos) [inline apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/_components/BreadcrumpInput.tsx:32]: now that you are at it fix the typo in the file **Breadcrumb**

**Review by geclos — CHANGES_REQUESTED** [review]
repository has an unsafe query, also review you haven't broken any modals by removing the layout component in evaluations. The rest looks fine 👌🏼

File: apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx (hunk 1 above)
Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx:38]: I think it's not
image

Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx:38]: Great catch 💯 but remove evaluation is already wrong in production

The button is in `/evaluations` but it goes to `/evaluations/:id/dashboard/destroy` which super weird weird
image

Comment (geclos) [reply apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx:38]: why? doesn't look weird to me at all

File: apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/_components/BreadcrumpInput.tsx (hunk 2 above)
Comment (andresgutgon) [reply apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/_components/BreadcrumpInput.tsx:32]: omg I did 1k times this typo 🙈 😂 🙈

File: apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx (hunk 1 above)
Comment (andresgutgon) [reply apps/web/src/app/(private)/evaluations/(evaluation)/layout.tsx:38]: Before from `/evaluations` -> `evaluations/:id/destroy`

Now it goes `/evaluations` -> `/evaluations/:id/destroy`

In the same layout so modal is over evaluations list
image

**Review by geclos — APPROVED** [review]

**Pull request merged**